modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -456,6 +454,7 @@ export default {
   },
   async mounted() {
     window.apos.adminBar.height = this.$refs.adminBar.offsetHeight;
+    window.apos.util.addClass(this.$refs.adminBarOuter, `apos-theme--primary-${window.apos.ui.theme.primary}`);
```

Comment:
Can't this be done with :class and a computed value?

Reply:
I'm thinking these should be in `created` instead. Get in there earlier. Assuming `$refs` is there then.

Reply:
Actually, if it's computed we no longer need the `ref`.

Reply:
now a computed via a theme mixin